### ISSUE TITLE
Newsletter subscription form (footer)

### ISSUE THREAD

**Issue by artur-rs** [3mdeb/3mdeb-website#176]
* is it working? I cannot find any parameters related to the selfhosted Mailtrain instance. As per description, I think it should add email to the https://newsletter.3mdeb.com/subscription/PW6XnCeK6, but doesn't work as expected.
* some sort of confirmation box should appear:
  - in the WP instance, footer section, there is ajaxdone:
  ```
  <p class="ajaxdone" style="font-size: 15px; color: #38d430; display: none;">You have subscribed successfully.</p>
  ```
  - in the blog instance, footer section, there is no such form as subscribe now

**Comment by Stanislaw-bnk** [comment]
You are right, the newsletter form seems not to work as expected.. I will look into it.

**Comment by Stanislaw-bnk** [comment]
@artur-rs I have commited a temporary solution: https://github.com/3mdeb/3mdeb-website/pull/184. It’s probably not the ideal way it should function, but at least it gets the job done for now.

**Comment by artur-rs** [comment]
Temporary solution is ok, we are migrating to other service anyway, so we will fix it then. I'll look into it in the free time

**Comment by artur-rs** [comment]
@Stanislaw-bnk merged, thank you for the contribution.
Prod PR: https://github.com/3mdeb/3mdeb-website/pull/187/files
